crypto/modes/asm/aes-gcm-avx512.pl

```diff
@@ -200,14 +200,7 @@
 # ;;; Helper functions
 # ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-# ; Generates "random" local labels
-sub random_string() {
-  my @chars  = ('a' .. 'z', 'A' .. 'Z', '0' .. '9', '_');
-  my $length = 15;
-  my $str;
-  map { $str .= $chars[rand(33)] } 1 .. $length;
-  return $str;
-}
+my $label_count = 0;
```

Comment:
(minor; not sure if it's worth another round to fix this; up to you) variable declaration and initialisation probably shouldn't be under "Helper functions"

Reply:
just saw this. making change now